x-pack/plugins/security_solution/public/actions/types.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CellAction, CellActionExecutionContext } from '@kbn/cell-actions';
+
+export interface SecurityCellActionExecutionContext extends CellActionExecutionContext {
+  metadata?: {
+    scopeId?: string; // needed in dataTables (events/alerts tables) and Timeline
```

Comment:
Brilliant idea! Also, I think, we should provide the documentation to explain the usage better.

Reply:
Yes, that would be helpful. Do you think a descriptive `TSDoc` in the **scopeId** would be enough? or should we create a new _x-pack/plugins/security_solution/public/actions/README.md_ to explain everything?